moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/useProjectListViewQuery.js

```diff
@@ -12,7 +12,7 @@ export const useGetProjectListView = ({
   advancedSearchWhereString,
 }) => {
   const { query, exportQuery } = useMemo(() => {
-    const query = gql`{
+    const query = gql`query ProjectListView {
```

Comment:
the `ProjectListView` doesn't do us much good because refetching by it's name doesn't work (see my note above), but it is a best practice to include it anyway.